content/en/synthetics/guide/authentication-protocols.md

```diff
@@ -16,7 +16,7 @@ further_reading:
 
 ## Overview
 
-[HTTP tests][1] allow you to send HTTP requests to your applications' API endpoints to verify responses and defined conditions, such as overall response time, expected status code, header, or body content. [Multistep API tests][2] allow you to chain HTTP requests to proactively monitor sophisticated journeys on your key services, and ensure they are available at any time and from any managed or private location.
+[HTTP tests][1] allow you to send HTTP requests to your applications' API endpoints to verify responses and defined conditions, such as overall response time, expected status code, header, or body content. [gRPC tests][8] allow you to monitor your gRPC services and servers with unary calls and health checks. [Multistep API tests][2] allow you to chain HTTP or gRPC requests to proactively monitor sophisticated journeys on your key services, and ensure they are available at any time and from any managed or private location.
```

Comment:
Since Basic Auth is not supported in gRPC tests, I didn't update anything else in the body of this guide, but if that assumption is false, please let me know or leave a suggestion!